src/components/Carousel/Carousel.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {StoryObj} from '@storybook/react-native';
+import {Dimensions, Image, StyleSheet, View} from 'react-native';
+import {Decorator} from '../../../.storybook/decorators';
+import {Carousel} from './Carousel';
+import {CarouselProps} from './Carousel.types';
+import {SearchCarouselData} from '../../assets/data';
+import {SearchCarouselRenderItem} from '../../screens/HomeScreen/components';
+
+const MyCarousel = {
+  title: 'Carousel',
+  component: Carousel,
+  decorators: [Decorator],
+};
+
+export default MyCarousel;
+
+type Story = StoryObj<CarouselProps>;
+
+export const DefaultCarousel: Story = {
+  args: {
+    data: SearchCarouselData,
+    width: Dimensions.get('window').width,
```

Comment:
Create Utils functions to fetch Device width and height instead of using Dimensions like this everywhere. Also, can we not use 100% or 50% instead?

Reply:
Yeah but carousel width and height are expecting a number type and when setting '100%' it is not working.